resources/js/privacy-modal.js

```diff
@@ -0,0 +1,29 @@
+window.addEventListener('DOMContentLoaded', function() {
+  document.querySelectorAll('.js-privacy-link').forEach(el => {
+    el.addEventListener('click', function(e) {
+      e.preventDefault();
+      document.getElementById('privacyModal').classList.remove('hidden');
+    });
+  });
+
+  const closeBtn = document.getElementById('closePrivacyModal');
+  if (closeBtn) {
+    closeBtn.addEventListener('click', function() {
+      document.getElementById('privacyModal').classList.add('hidden');
+    });
+  }
+  
+  const closeIcon = document.getElementById('closePrivacyModalIcon');
+  if (closeIcon) {
+    closeIcon.addEventListener('click', function() {
+      document.getElementById('privacyModal').classList.add('hidden');
+    });
+  }
```

Comment:
@baovnt-zigexn 
Nên thay bằng '?' để gọn code.
```suggestion
    closeIcon?.addEventListener('click', function() {
      document.getElementById('privacyModal').classList.add('hidden');
    });
```